language/e2e_tests/src/account_universe.rs

```diff
@@ -82,16 +82,16 @@ pub trait AUTransactionGen: fmt::Debug {
     /// necessary. Returns a signed transaction that can be run on the VM and the expected output.
     fn apply(&self, universe: &mut AccountUniverse) -> (SignedTransaction, TransactionStatus);
 
-    /// Creates a boxed version of this transaction, suitable for dynamic dispatch.
-    fn boxed(self) -> Box<dyn AUTransactionGen>
+    /// Creates an arced version of this transaction, suitable for dynamic dispatch.
+    fn arced(self) -> Arc<dyn AUTransactionGen>
```

Comment:
did we need to move to Arc because we are now running over multiple genesis in parallel?

Reply:
We aren't really running them in parallel. The issue is that we need to clone the Gen traited object and run them on different genesis and Arc seems to be the only way for us to be able to clone it safely from Rust's stand point.